tests/unit/jaml/test_type_parse.py

```diff
@@ -179,3 +180,11 @@ def test_parsing_brackets_in_envvar():
         assert b['executors'][0]['env']['var2'] == 'a'
         assert b['executors'][0]['env']['var3'] == '{"1": "2"}-a'
         assert b['executors'][0]['env']['var4'] == '-{"1": "2"}'
+
+
+def test_exception_invalid_yaml():
+    with pytest.raises(BadConfigSource):
+        BaseExecutor.load_config('./invalid.yml')
```

Comment:
do not use `./invalid.yml`, use some `cur_dir` to make sure test is runnable from anywhere